recipes/berokka/meta.yaml

```diff
@@ -1,31 +1,37 @@
+{% set name = "berokka" %}
+{% set user = "tseemann" %}
```

Comment:
These are rather excessive unless you plan on renaming the package and transferring ownership.

Reply:
All my existing packages in bioconda use this.  I understand your argument. I would love to transfer the ownership one day.  Is there any bioconda policy against me doing this?

Reply:
They're your projects, so we certainly have nothing against you transferring them if/when you want :)